Galgenmännchen.py

```diff
@@ -0,0 +1,77 @@
+from hangmanStages import stages
+from Words import word_list
+import random
+
+
+def PrintHangmanStage():
+    print(stages[6])
+
+
+def GetRandomWordfromList():
+    global word
+    word = random.choice(word_list).upper()
+
+
+def GetWordIndices(word, guess):
+    indices = []
+    for index, letter in enumerate(word):
+        if letter == guess:
+            indices.append(index)
+
+    return indices
+
+
+def Play():
+
+    PrintHangmanStage()
+    print("Willkommen zu Galgenmännchen")
+    GetRandomWordfromList()
+    word_was_guessed = False
+    remaining_tries = 6
+    hidden_letter_symbol = "_"
```

Comment:
Could be a constant?

Reply:
This Variables have to be inside the function and they are only used in this function so I think in this way it makes more sense